WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -0,0 +1,144 @@
+                + "user_name               TEXT,"
+                + "first_name              TEXT,"
+                + "last_name               TEXT,"
+                + "display_name            TEXT,"
+                + "avatar_url              TEXT,"
+                + "role                    TEXT,"
+                + "PRIMARY KEY (person_id, local_blog_id)"
+                + ");");
+    }
+
+    private static void dropTables(SQLiteDatabase db) {
+        db.execSQL("DROP TABLE IF EXISTS " + PEOPLE_TABLE);
+    }
+
+    public static void reset(SQLiteDatabase db) {
+        AppLog.i(AppLog.T.PEOPLE, "resetting people table");
+        dropTables(db);
+        createTables(db);
+    }
+
+    public static void save(Person person) {
+        save(person, getWritableDb());
+    }
+
+    public static void save(Person person, SQLiteDatabase database) {
+        ContentValues values = new ContentValues();
+        values.put("person_id", person.getPersonID());
+        values.put("blog_id", person.getBlogId());
+        values.put("local_blog_id", person.getLocalTableBlogId());
+        values.put("user_name", person.getUsername());
+        values.put("first_name", person.getFirstName());
+        values.put("last_name", person.getLastName());
+        values.put("display_name", person.getDisplayName());
+        values.put("avatar_url", person.getAvatarUrl());
+        values.put("role", person.getRole());
+        database.insertWithOnConflict(PEOPLE_TABLE, null, values, SQLiteDatabase.CONFLICT_REPLACE);
+    }
+
+    public static void savePeople(List<Person> peopleList, int localTableBlogId) {
+        getWritableDb().beginTransaction();
+        try {
+            //We have a fresh list, remove the previous list of people in case it was deleted on remote
+            PeopleTable.deletePeopleForBlog(localTableBlogId);
+
+            for (Person person : peopleList) {
+                PeopleTable.save(person);
+            }
+            getWritableDb().setTransactionSuccessful();
+        } finally {
+            getWritableDb().endTransaction();
+        }
+    }
+
+    public static void deletePeopleForBlog(int localTableBlogId) {
+        String[] args = new String[]{Integer.toString(localTableBlogId)};
+        getWritableDb().delete(PEOPLE_TABLE, "local_blog_id=?", args);
+    }
+
+    public static void deletePerson(long personID, int localTableBlogId) {
+        String[] args = new String[]{Long.toString(personID), Integer.toString(localTableBlogId)};
+        getWritableDb().delete(PEOPLE_TABLE, "person_id=? AND local_blog_id=?", args);
+    }
+
+    public static List<Person> getPeople(int localTableBlogId) {
+        List<Person> people = new ArrayList<>();
+        String[] args = { Integer.toString(localTableBlogId) };
+        Cursor c = getReadableDb().rawQuery("SELECT * FROM " + PEOPLE_TABLE +
+                " WHERE local_blog_id=? ORDER BY display_name, user_name", args);
```

Comment:
This uses case-sensitive sorting, so "Nick Bradbury" and "nick bradbury" don't appear next to each other in the people list. I think this should be case-insensitive like the web.